packages/core/src/visitors.ts

```diff
@@ -380,8 +393,11 @@ export function normalizeVisitors<T extends BaseVisitor>(
     }
 
     for (const typeName of visitorKeys as Array<keyof T>) {
-      const typeVisitor = visitor[typeName] as any as NestedVisitObject<any, T>;
-      const normalizedTypeVisitor = normalizedVisitors[typeName]!;
+      const typeVisitor = (visitor[typeName] ||
```

Comment:
do we have a test for a legacy names somewhere? if not we should add temporary.

Reply:
Good point. Will add a them.